lib/kerosene/html/simple.ex

```diff
@@ -11,4 +11,13 @@ defmodule Kerosene.HTML.Simple do
 
   defp build_html_class(true), do: "active"
   defp build_html_class(_), do: nil
+
+  defp wrapper_opts(additional_class) do
```

Comment:
Do you mean allowing `opts[:class]` and something like`opts[:current_link_class]` for `Kerosene.HTML.paginate` function so we can also customize current page link?